pages/1.14/administering-clusters/draining-a-node/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: layout.pug
+navigationTitle:  Draining a Node
+title: Draining a Node
+menuWeight: 801
+excerpt: Draining an agent node in an active DC/OS cluster
+enterprise: false
+render: mustache
+model: /1.14/data.yml
+---
+
+You can drain tasks from an agent node in an active DC/OS cluster in order to take the node down for maintenance or to remove it permanently from the cluster. Nodes may be drained via the DC/OS CLI, and progress can be monitored via the CLI or UI.
```

Comment:
Nodes will also be able to be drained via UI by them time the release goes GA. It didn't make the cut for alpha.